usecases/wip.md

```diff
@@ -0,0 +1,94 @@
+# Workflows
+> [Back](../specification.md)
+
+(This probably should be nice graphics but well ... here are some lists ...)
+
+## Party creation and joining
+> A User creates a new party, invites friends which join the party and starts the game.
+
+- User1 sends `CreatePartyEvent`
+    - Server creates a new Party (Local container in which all websocket connections are hold which belong to a party)
+    - Server sends `PartyCreatedEvent` to User1
+- User1 receives `PartyCreatedEvent` 
+    - User1 stores JWT locally
+- User1 shares partyId with User2
+- User2 joins party with `JoinPartyEvent` with partyId and a username
+    - Server recvieves event and creates a JWT for the user
+    - Server puts the connection into the container assicated with the party
+    - server sends `PartyJoinedEvent` to User2
+- User2 recvieves `PartyJoinedEvent`
+    - User2 stores JWT locally
+    - User2 waits for the game to be started
+- User1 start the game with `StartGameEvent`
+    - Server creates a new GameState for the current party.
+    - A `GameStateEvent` is sent to each user containing the GameState for the specific user
+        - (The activeUser object in the `GameStateEvent` is null/undefined. This indicates the start-voting-mechanism)
+
+
+## Player Voting to start
```

Comment:
As far as i understood the rulebook, there has to be a decision. There probably should be a possibility to trigger a new voting process. I guess extracting the voting process into separate events instead of a GameStateEvent with empty activePlayer should do the trick. 

So i would introduce a new Event, "StartVotingEvent"  with a unique identifier and the options available to vote for.

This would, in theory, allow for more vote-mechanisms like vote-kick, ...

Does it make sense to add a new API scope for the voting mechanism? So we have 4 (game,chat,party and vote)

Reply:
I think an additional scope would be nice for separation. We could also set a timeout after which the vote will be closed and a random person is chosen.

You could also start with a simple placeholder and have someone else work on that later 😁

Reply:
Had some spare time, added it ^^.

I actually don't think anyone else can edit this PR ...not sure